preset/deep-learning/docker-compose.yaml

```diff
@@ -54,6 +54,119 @@ services:
+        dependency.python: ${PYTHON_VERSION:-3.10}
+        dependency.python.pip: requirements.txt
+        dependency.apt.apt-utils: true
+        dependency.apt.build-essential: true
+        dependency.apt.bzip2: true
+        dependency.apt.ca-certificates: true
+        dependency.apt.clinfo: true
+        dependency.apt.cmake: true
+        dependency.apt.diffutils: true
+        dependency.apt.g++: true
+        dependency.apt.gcc: true
+        dependency.apt.git: true
+        dependency.apt.gnupg2: true
+        dependency.apt.gpg-agent: true
+        dependency.apt.gzip: true
+        dependency.apt.hwinfo: true
+        dependency.apt.intel-igc-cm: true
+        dependency.apt.intel-level-zero-gpu: "${LEVEL_ZERO_GPU_VER}"
+        dependency.apt.intel-media-va-driver-non-free: true
+        dependency.apt.intel-opencl-icd: "${ICD_VER}"
+        dependency.apt.level-zero: "${LEVEL_ZERO_VER}"
+        dependency.apt.level-zero-dev: "${LEVEL_ZERO_DEV_VER}"
+        dependency.apt.libegl1-mesa: true
+        dependency.apt.libegl1-mesa-dev: true
+        dependency.apt.libegl-mesa0: true
+        dependency.apt.libgbm1: true
+        dependency.apt.libgl1-mesa-dev: true
+        dependency.apt.libgl1-mesa-dri: true
+        dependency.apt.libglapi-mesa: true
+        dependency.apt.libgles2-mesa-dev: true
+        dependency.apt.libglx-mesa0: true
+        dependency.apt.libigc-dev: true
+        dependency.apt.libigdfcl-dev: true
+        dependency.apt.libigdgmm12: true
+        dependency.apt.libigfxcmrt-dev: true
+        dependency.apt.libmfx1: true
+        dependency.apt.libmfxgen1: true
+        dependency.apt.libopenmpi-dev: true
+        dependency.apt.libvpl2: true
+        dependency.apt.libxatracker2: true
+        dependency.apt.make: true
+        dependency.apt.mesa-va-drivers: true
+        dependency.apt.mesa-vdpau-drivers: true
+        dependency.apt.mesa-vulkan-drivers: true
+        dependency.apt.numactl: true
+        dependency.apt.openmpi-bin: true
+        dependency.apt.openmpi-common: true
+        dependency.apt.openssh-client: true
+        dependency.apt.openssh-server: true
+        dependency.apt.patch: true
+        dependency.apt.rsync: true
+        dependency.apt.sudo: true
+        dependency.apt.unzip: true
+        dependency.apt.va-driver-all: true
+        dependency.apt.vainfo: true
+        dependency.apt.wget: true
+        dependency.apt.xz-utils: true
+        dependency.conda.jupyterlab: '>=4.1.8'
+        dependency.conda.aiohttp: '>=3.9.0'
+        dependency.conda.cryptography: '>=42.0.4'
+        dependency.conda.dataset_librarian: '>=1.0.4'
+        dependency.conda.deepspeed: "${DEEPSPEED_VERSION}"
+        dependency.conda.dpcpp_impl_linux-64: "${IDP_VERSION}"
+        dependency.conda.dpcpp-cpp-rt: "${IDP_VERSION}"
+        dependency.conda.dpnp: "${NUMBA_DPEX_VERSION:-0.14.0}"
+        dependency.conda.idna: '>=3.7'
+        dependency.conda.impi-devel: "${IMPI_VERSION}"
```

Comment:
If this variable doesn't have a default version it just becomes `impi-devel`

Reply:
fixed